ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -535,7 +535,7 @@ default_interfaces:
     mlag_peer_l3_vlan: < 0-4094 | false | default -> 4093 >
 
     # IP address pool used for MLAG underlay L3 peering | *Required when MLAG leafs present in topology.
-    # IP is derived from the node id.
+    # IP is derived from the node id. Assign a subnet that can accomodate at least the total number of MLAG devices*2.
```

Comment:
After thinking a bit about this could we specify that this is needed for "the default allocation scheme in eos_designs" (probably worded a bit differently). If people use custom template for MLAG allocation (e.g. same /31 for every MLAG pair) then this comment is not relevant.

Reply:
Good point, and in the future, we have discussed supporting various algorithms for MLAG IP addressing.